integrationtests/Paket.IntegrationTests/PackSpecs.fs

```diff
@@ -349,3 +349,19 @@ let ``#1596 pack works for reflected definition assemblies``() =
     
     File.Exists(Path.Combine(outPath, "lib", "net45", "Project2.dll")) |> shouldEqual true
     File.Delete(templatePath)
+
+[<Test>]
+let ``#1806 pack localized dll`` () =
+    let scenario = "i001806-pack-localized"
+    let rootPath = scenarioTempPath scenario
+    let outPath = Path.Combine(rootPath, "out")
+    let package = Path.Combine(outPath, "LocalizedLib.1.0.0.0.nupkg")
+    
+    paket ("pack -v output \"" + outPath) scenario |> ignore
+    ZipFile.ExtractToDirectory(package, outPath)
+
+    Path.Combine(outPath, "lib", "net45", "LocalizedLib.dll") |> checkFileExists
```

Comment:
While it looks sweet I have not seen it in the rest of the code so not sure. Trying to stay consistent with the codebase. Also as a noob `Path.Combine(...)` is nice as I know what it does.